pkg/metrics/statsd/k8s/metrics.go

```diff
@@ -48,4 +48,9 @@ func (t *tracer) Increment(code string, verb string, host string) {
 		"verb": verb,
 		"code": code,
 	})
+	if code == "<error>" {
```

Comment:
@jim-minter  Wasn't sure exactly what this would look like.  I found an example of usage here https://github.com/Azure/ARO-RP/blob/master/vendor/k8s.io/client-go/rest/request.go#L627 and took a guess.

Reply:
Looks plausible, thanks